server/selinux/server/pulp-streamer.te

```diff
@@ -21,6 +21,9 @@ require {
 }
 
 
+unconfined_shell_domtrans(streamer_t)
```

Comment:
This suggests that the streamer is running as unconfined_t and then calling to a shell. What process context is the streamer running as?

Reply:
Which shell is this executing that needs to transition to unconfined_t?